index.html

```diff
@@ -172,9 +172,41 @@ <h1>My Recent Works</h1>
   <ul class="See-btn">
     <li><button type="button" class="see-project">See Project</button></li>
   </ul>
-</li>
+</li> -->
 </ul>
 </section>
+
+<section class="mobilePopup" id="firstPopup">
+  <!-- <ul>
+  <li class="flex-box1">
+    <ul class="firstListPop">
+      <li><span class="cls fa fa-close" id="firstClose"></span></li>
+      <li><img src="./Images/SecondPortfolio.svg" alt="My-Work"></li>
+      <li><h1>Keeping track of hundreds of
+        components</h1></li>
+    </ul>
+    <div class="list-boxPop">
+    <ul>
+    <li>Ruby on rails</li>
+    <li>css</li>
+    <li>JavaScript</li>
+    </ul>
+    </div>
+    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+     Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+      when an unknown printer took a galley of type
+      and scrambled it 1960s with the releaLorem Ipsum is simply dummy text
+      of the printing and typesetting industry. Lorem Ipsum has been the
+      industry's standard dummy text ever since the 1500s,
+     when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+  <ul class="See-btn1">
+    <li><button type="button" class="see-project">See Live<i class="fa fa-external-link" aria-hidden="true"></i></button></li>
+    <li><button type="button" class="see-project">See Source<i class="fa fa-github" aria-hidden="true"></i></button></li>
+  </ul>
+</li>
+</ul> -->
```

Comment:
- [ ] Commenting code is not a good way when you are sending for production, it usually use for testing purpose and should deleted while sending for production. Please remove all the commented code to make your code more cleaner and nicer.